src/character.cpp

```diff
@@ -6359,7 +6364,8 @@ bool Character::has_calorie_deficit() const
 
 units::mass Character::bodyweight() const
 {
-    return bodyweight_fat() + bodyweight_lean();
+    return std::max( 1_gram, enchantment_cache->modify_value( enchant_vals::mod::WEIGHT_MULT,
+                     normal_weight ) );
```

Comment:
`Assuming normal_weight is an actual var`
```suggestion
                     bodyweight_fat() + bodyweight_lean() ) );
```